IVehicle.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CodeReviewDemo
+{
+    internal interface IVehicle
+    {
+        void StartEngine();
+        void StopEngine();
+        void Accelerate();
+        void Brake();
+
+        void PrepareCoffee();
+        void AddCoffee();
+    }
+}
```

Comment:
The methods `PrepareCoffee` and `AddCoffee` seem out of place in the `IVehicle` interface. Interfaces should define a contract relevant to their purpose. It is recommended to remove these methods from `IVehicle` and consider if they belong to another interface more relevant to their functionality.